### ISSUE TITLE
Divergency between TTS text and phrase in Test voice prompt dialog

### ISSUE THREAD

**Issue by naoliv** [osmandapp/Osmand#3735]
In https://github.com/osmandapp/OsmAnd-resources/blob/master/voice/en/ttsconfig.p#L88 we have:

```prolog
% NEARBY POINTS
string('and_arrive_waypoint.ogg', 'and pass GPX waypoint ').
string('reached_waypoint.ogg', 'you are passing GPX waypoint ').
string('and_arrive_favorite.ogg', 'and pass favorite ').
string('reached_favorite.ogg', 'you are passing favorite ').
string('and_arrive_poi.ogg', 'and pass POI ').
```

But when testing the sounds we can see that instead having messages with **pass** with have messages with **arrive**:
![](https://i.imgur.com/hX7HFap.png)

It seems that originally the message used _arrive_ in every phrase, instead _pass_.

What is the correct meaning here? arrive or pass?

If pass is the proper word, then the text in the _Test voice prompt_ dialog should be updated to reflect the proper usage.

**Comment by sonora** [comment]
Thanks for pointing this out!

Without doing a deep dive here, I am pretty certain from my recollection things are as follows: Our original wording was always "arrive" for both
- **destination and intermediate destination**, where it is correct and makes sense,
- but also for **POI, GPX waypoint, and Favorites**.

But in the latter case we actually trigger these messages when within a certain radius of the point in question (while they are usually never directly **on** your path). So (contrary to destinations) these point may not even be in sight, and we changed the wording for them from "arrive" to the more adequate "pass".

Long story short: Our productive code in the ttsconfg wording file is correct. But I had missed to adjust the display text in the Test Voice Prompt code accordingly. Will do so now, and close this.
